{{ cookiecutter.github_repo_name }}/tests/test_calculator.py

```diff
@@ -1,35 +1,70 @@
+import numpy as np
 import pytest
-from {{cookiecutter.package_dir_name}} import calculator
+from {{cookiecutter.package_dir_name}} import calculator as calc
 
 
-def test_dot_product_2D():
-    """Test the dot product function with 2D vectors."""
+def test_dot_product_2D_list():
     a = [1, 2]
     b = [3, 4]
-    expected = 11
-    actual = calculator.dot_product(a, b)
+    expected = 11.0
+    actual = calc.dot_product(a, b)
     assert actual == expected
 
 
-def test_dot_product_3D():
-    """Test the dot product function with 3D vectors."""
+def test_dot_product_3D_list():
     a = [1, 2, 3]
     b = [4, 5, 6]
-    expected = 32
-    actual = calculator.dot_product(a, b)
+    expected = 32.0
+    actual = calc.dot_product(a, b)
+    assert actual == expected
+
+def test_dot_product_2D_tuple():
```

Comment:
We don't need these because it is all tested with `pytest.mark.parametrize` below.  We don't want to model incorrect behavior.  I think it is good to write a test that doesn't use paramatrize, but just one simple test that does on thing as an example.

Reply:
Right. We don't want to mislead the user. Just 2 raw example above, as suggested.